docs/src/components/code-examples.js

```diff
@@ -1,9 +1,123 @@
 import React from 'react'
-import Code from './code'
+import styled, { css } from 'styled-components'
+import { StaticQuery, graphql } from 'gatsby'
+import SyntaxHighligher from './syntax-highlighter'
 
-export default props => (
-  <div>
-    <div>code examples for {props.subject}</div>
-    <Code language="python">lambda x: x + 1</Code>
-  </div>
+const linkCSS = css`
+  line-height: 1.25;
+  padding: 1.25rem 1.5rem;
+  border-bottom: ${props =>
+    props.active ? '0.35rem solid #007099' : '0.35rem solid #e0e0e0'};
+  text-decoration: none;
+  display: block;
+  color: #007099;
+
+  &:hover {
+    color: #000;
+    cursor: pointer;
+  }
+`
+
+const Tabs = styled.div`
+  margin-top: 1rem;
+  border-top: 1px solid #cfcfcf;
+`
+
+const TabsNav = styled.div`
+  border-bottom: 0.35rem solid #e0e0e0;
+  width: 100%;
+`
+
+const Ul = styled.ul`
+  margin: 0 0 -0.35rem;
+  padding: 0;
+  display: inline-block;
+  line-height: 1.5;
+  list-style-type: none;
+`
+
+const NavItem = styled.li`
+  margin: 0;
+  padding: 0;
+  display: inline-block;
+`
+
+const NavLink = styled.label`
+  ${linkCSS};
+`
+
+const TabContent = styled.div`
+  display: ${props => (props.active ? 'block' : 'none')};
+`
+
+export const filterTransformSortCodeSnippets = ({ codeSnippets, path }) => {
+  if (!codeSnippets) {
+    return []
+  }
+
+  return codeSnippets
+    .filter(n => n.node.relativePath.startsWith(path))
+    .map(n => n.node)
+    .sort((a, b) => a.name.localeCompare(b.name))
+}
+
+export class CodeExamplesComponent extends React.Component {
+  state = {
+    activeTab: 0,
+  }
+
+  handleClick = index => {
+    this.setState({ activeTab: index })
+  }
+
+  render = () => {
+    const codeSnippets = filterTransformSortCodeSnippets({
+      codeSnippets: this.props.data.allCodeSamples.edges,
+      path: this.props.path,
+    })
+    return (
+      <Tabs>
+        <TabsNav>
+          <Ul>
+            {codeSnippets.map((s, i) => (
+              <NavItem key={i}>
+                <NavLink
```

Comment:
Pretty sure we want this to be an `a`nchor element rather than a label. E.g. see https://designsystem.gov.au/components/buttons/ and tab through the page - aside from what they do, I'd expect to be able to tab through the page and when I get to "Python" or "Go", for example, to hit 'enter' and have it change to that tab.